ouroboros-consensus-byron-test/test/Test/ThreadNet/RealPBFT.hs

```diff
@@ -1288,15 +1291,18 @@ genNodeRekeys params nodeJoinPlan nodeTopology numSlots@(NumSlots t)
 mkRekeyUpd
   :: Genesis.Config
   -> Genesis.GeneratedSecrets
+  -> EpochInfo Identity
+  -> CoreNodeId
   -> ProtocolInfo (ChaChaT m) ByronBlock
-  -> EpochNo
+  -> SlotNo
   -> Crypto.SignKeyDSIGN Crypto.ByronDSIGN
   -> Maybe (TestNodeInitialization m ByronBlock)
-mkRekeyUpd genesisConfig genesisSecrets pInfo eno newSK =
+mkRekeyUpd genesisConfig genesisSecrets epochInfo _ pInfo sno newSK =
   case pInfoLeaderCreds of
     Nothing              -> Nothing
     Just (isLeader, mfs) ->
       let PBftIsLeader{pbftCoreNodeId} = isLeader
+          eno = runIdentity $ epochInfoEpoch epochInfo sno
```

Comment:
On some level, this should be `futureSlotToEpoch` instead of `fixedSizeEpochInfo`. But that will only be necessary if `RealPBFT` is updated to include a hard fork. So it's moot until then.